### DISCUSSION TITLE
Weekly Sync #27 - Jul 07, 2021

### DISCUSSION THREAD

**Discussion by andresin87** [SUI-Components/sui-components#1600] [Weekly Sync Streaming]
[Notes from our last Weekly meeting](https://github.com/SUI-Components/sui-components/discussions/1585)

**Agenda**

- useSunscreen()
- [Dashboard](https://pages.github.mpi-internal.com/scmspain/design-systems/sui/index.html)
- Iteration Atom Switch to make tests easily
- Upcoming
  - [Skeleton](https://sui-components.vercel.app/workbench/atom/skeleton/demo)
  - [BackToTop](https://sui-components.vercel.app/workbench/atom/backToTop/demo) behaviour bugs n fixes
  - [Avatar](https://sui-components.vercel.app/workbench/molecule/avatar/demo) next steps
 - New components in [Docs-Components](https://github.com/SUI-Components/docs-components)
- New component [Validation code](https://github.com/SUI-Components/sui-components/discussions/1574)
 - [Sui Flexibility discussion](https://github.com/SUI-Components/sui-components/discussions/1588)

**Comment by oriolpuig** [comment]
Do you mean `useSunscreen()`? @andresin87

**Reply by nistadev** [reply]
@oriolpuig you forgot weather check: 
```javascript
if(!Today.isCloudy()) {
  useSunscreen()
}
```

**Reply by turolopezsanabria** [reply]
```javascript
else {
      useSunscreen()
    }
```

**Reply by andresin87** [reply]
you cannot `useHook()` in a conditional statement

**Comment by a-ferrante** [comment]
Iteracion del AtomSwitch para poder hacer test con la testing library mas facilmente
